index.html

```diff
@@ -232,4 +232,36 @@ <h3>Skills</h3>
             </div>
         </section>
     </section>
+    <section class="footer">
+        <div class="header-img-left">
+            <img src="images/image_geometry_1.png" alt="image">
+        </div>
+        <div class="contact">
+            <div class="contact-description">
+                <h3>Get started</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            </div>
+            <form action="https://formspree.io/f/xbjwgkqr" method="post">
+                <input type="text" name="name" id="name" placeholder="Kiprono" required>
+                <input type="email" name="email" id="email" placeholder="|" value="Your email" required>
+                
+                <textarea name="message" id="msg" cols="30" rows="10"></textarea>
+                <button class="btn-collabo" type="submit" name="submit">Start collaboration</button>
+
+            </form>
+        </div>
+        <div class="social-footer">
+            <div class="social-container">
+                <p>Get my Resume <span><img src="images/resume.png" alt="image"/></span></p>
+                <ul>
+                    <li><img src="images/stackoverflow.png" class="fa-img" alt="image"></li>
```

Comment:
- [ ] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your footer design appears differently from the template. Please fix your design and make sure it matches the template.

Your design

![Screenshot (165)](https://user-images.githubusercontent.com/83803414/155012523-1835424b-5f49-4172-abbc-f281fae3e37c.png)

Figma design

![Screenshot (164)](https://user-images.githubusercontent.com/83803414/155012540-ee89cc2c-8c91-4a19-b673-bbbf687944f8.png)

Reply:
resolved